package.json

```diff
@@ -50,11 +32,9 @@
     "@excalidraw/prettier-config": "1.0.1",
     "@typescript-eslint/eslint-plugin": "4.15.2",
     "@typescript-eslint/parser": "4.15.1",
-    "babel-eslint": "10.1.0",
-    "babel-loader": "8.2.2",
```

Comment:
do we need those?

Reply:
babel-loader, [yes](https://github.com/excalidraw/svg-to-excalidraw/blob/main/webpack.config.js#L6), babel-eslint, seems not.